api/openshift/api.dc.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
   selfLink: ''
   name: biohubbc-api-dc
 parameters:
+  - name: SHOULD_CLAMAV_SCAN
+    value: '1'
```

Comment:
Do we want the default to be `true` instead of `1`?

In your check in file-utils.ts, you could then check if `process.env.SHOULD_CLAMAV_SCAN === 'true'`

Reply:
This way, someone setting it to `false` will still result in it being disabled, even those its a truthy string.